### PR TITLE
Add metadata.Country.iso_alpha2_code field

### PR DESCRIPTION
**Pull request by marcofucci**
### Description of change

This:
- adds the field `metadata.Country.iso_alpha2_code`
- makes `Country` and `OverseasRegion` readonly in the admin
- removes some invalid countries from constants
- replaces some invalid countries in `test_data`
- reformats `countries.yaml` after removing the invalid countries

Still to be solved:
- the `countries.yaml` should really be loaded by the `0001` metadata migration. It's not because using `loaddata` in a migration with a different data structure breaks things
- the iso data will be loaded using a different PR

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [x] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: changelog/iso.db [hunk 1]
```diff
@@ -0,0 +1 @@
+A new field `iso_alpha2_code` was added to the `metadata_country` table. It has not been populated yet.
```
Comment (reupen) [inline changelog/iso.db]: Should add some entries for the other changes as well (read-only admin, updated initial country list etc.)

File: datahub/metadata/admin.py [hunk 2]
```diff
@@ -96,12 +96,11 @@ class EditableOrderOnlyOrderedMetadataAdmin(OrderedMetadataAdmin, ViewAndChangeO
 
 
 @admin.register(models.Country)
-class CountryAdmin(admin.ModelAdmin):
+class CountryAdmin(ViewOnlyAdmin):
     """Admin for countries."""
 
     fields = ('pk', 'name', 'overseas_region', 'disabled_on')
```
Comment (reupen) [inline datahub/metadata/admin.py]: Are you going to add the ISO code here later?

**Comment by reupen** [comment]
@marcofucci Couple of other points.

1. Not for now, but I was wondering if we would need a 'secondary ISO codes' field since these countries won't cover all of them (mainly depending on what ISO codes are used in data coming in from other sources). This could be added when we actually need it, so just something to think about.

2. Have you mentioned this to Anthony? I see we have null for some overseas_regions, which isn't ideal (they may be countries that we don't need, though).

File: changelog/iso.db (hunk 1 above)
Comment (marcofucci) [reply changelog/iso.db]: yeah maybe one for the readonly admin, the initial country list is just a snapshot of the current live one so it does not change any behaviour.

File: datahub/metadata/admin.py (hunk 2 above)
Comment (marcofucci) [reply datahub/metadata/admin.py]: I was going to add it later as they will be empty but I might forget so, I could add it now...

**Comment by marcofucci** [comment]
@reupen 
> Not for now, but I was wondering if we would need a 'secondary ISO codes' field since these countries won't cover all of them (mainly depending on what ISO codes are used in data coming in from other sources). This could be added when we actually need it, so just something to think about.

I think we should have one to one mapping, I'm waiting to get access to DnB metadata to see how different their list is from ours but I think it would be safer to include the ones they have. I diffed the list we have in prod with the FCO and there are a few extra countries in Data Hub not in the FCO list but in the ISO one already anyway.

> Have you mentioned this to Anthony? I see we have null for some overseas_regions, which isn't ideal (they may be countries that we don't need, though).

The PR is not replacing any data, the version in `countries.yaml` is just the latest from production. I replaced it because it's good to have that field populated in new environments and to get rid of the invalid countries we had before.

File: changelog/iso.db (hunk 1 above)
Comment (reupen) [reply changelog/iso.db]: The data science guys use the fixtures for some stuff, so might still be worth mentioning.

**Comment by marcofucci** [comment]
@reupen Anthony and I went through a few countries, assigned the overseas regions and disable some of them. Thanks for the heads up!

File: fixtures/metadata/countries.yaml [hunk 3]
```diff
@@ -133,9 +133,9 @@
 - model: metadata.country
   pk: 1a50bdb8-5d95-e211-a939-e4115bead28a
   fields:
-    disabled_on: null
-    name: Netherlands Antilles
-    overseas_region: null
+    disabled_on: '2018-11-05T17:00:00Z'
+    name: Netherlands Antilles,
```
Comment (reupen) [inline fixtures/metadata/countries.yaml]: Is this comma meant to be there?

File: fixtures/metadata/countries.yaml [hunk 4]
```diff
@@ -1,744 +1,1308 @@
...
+    disabled_on: null
+    name: Saudi Arabia
+    overseas_region: c4679b44-079e-4394-8bf7-bb0881a5031d
 - model: metadata.country
   pk: 1a50bdb8-5d95-e211-a939-e4115bead28a
-  fields: {name: Netherlands Antilles}
+  fields:
+    disabled_on: '2018-11-05T17:00:00Z'
+    name: Netherlands Antilles,
+    overseas_region: 3e6809d6-89f6-4590-8458-1d0dab73ad1a}
```
Comment (reupen) [inline fixtures/metadata/countries.yaml]: And the closing brace here...

**Pull request merged**